lib/main.js

```diff
@@ -13,6 +13,17 @@ let tslintDef;
 let requireResolve;
 const idleCallbacks = new Set();
 
+async function statAsync(pathname) {
```

Comment:
Maybe name this just `stat` to keep inline with the [api](https://nodejs.org/api/fs.html#fs_fs_stat_path_callback)